snapshot/cli/cli.go

```diff
@@ -229,9 +231,17 @@ func (c *createGitBundleCommand) run(db snapshot.DB, _ *cobra.Command, _ []strin
 		}
 		tempDir = td.Dir
 
+		// Get the length of this revList for the sha we will compute for file name
```

Comment:
We need some extra 'entropy' because our given revList as <rev>..<ref> which is in practice "<rev>..master" is not unique - there could be different numbers of things inbetween <rev> and master.  In fact this was causing collisions with funnel bundles where a TIP bundle created with a recent <rev> eventually became the basis rev for the AUX bundle, and resulted in the same generated sha being used which caused some init failures.